### PR TITLE
issue #725 changes to ResourceAccessed.js

### PR DESCRIPTION
**Pull request by zqian**
the current approach uses the useSetUserSetting.js, similar to other two views.

related the [issues-642](https://github.com/tl-its-umich-edu/my-learning-analytics/issues/642)

### CODE REVIEW

File: assets/src/containers/ResourcesAccessed.js [hunk 1]
```diff
@@ -104,32 +104,25 @@ function ResourcesAccessed (props) {
...
+          },
+          credentials: 'include',
+          body: JSON.stringify({resource: gradeRangeFilter}),
+          method: 'PUT'
+        }).then(handleError)
+          .then(res => res.json())
+          .then(res => {
+            setResponse(res)
+            setLoaded(true)
+        }).catch(error => setError(error.message))
```
Comment (ssciolla) [inline assets/src/containers/ResourcesAccessed.js]: Is `setError` available here? It seems to be defined in `useFetch.js` and then not defined or imported within this file.

Comment (ssciolla) [reply assets/src/containers/ResourcesAccessed.js]: Yeah, I may be missing something; I'm not super knowledgeable about hooks. I guess I have a similar question about `setResponse` and `setLoaded`.

Comment (justin0022) [reply assets/src/containers/ResourcesAccessed.js]: None of them are imported or available - they are all undefined. 

I recommend using [standard](https://standardjs.com/), a JavaScript linter. It'll reveal any undefined functions/variables.

Comment (zqian) [reply assets/src/containers/ResourcesAccessed.js]: OK. I made another iteration to use useSetUserSetting.js. Please review.

**Comment by justin0022** [comment]
I'm not seeing the week range slider in my view:
Screen Shot 2019-10-01 at 9 03 24 AM
 
I was actually working on the refactor of this view a couple of weeks ago, but I switched to working on the Assignment Planning redesign. For what it's worth, here's the progress I made to refactoring this view: https://github.com/justin0022/my-learning-analytics/blob/refactor-resources/assets/src/containers/ResourcesAccessed.js

**Comment by pushyamig** [comment]
@justin0022 The problem seems to be caused `get_user_default_selection` called each time after the set_user_default_selection. I am not sure what changed in between it was not the behavior before. I am taking a look at it. this may be due to the way `useFetch` is implemented now not sure. This is not happening with Assignmentview since the refactored it

**Comment by pushyamig** [comment]
Closing as this is fixed  in #735

**Pull request closed without merging**